.pre-commit-config.yaml

```
@@ -17,11 +17,11 @@ repos:
           ]
 
   - repo: https://github.com/psf/black
-    rev: 23.3.0
+    rev: 23.11.0
```

```suggestion
    rev: 22.6.0
```
Aligning with dev-requirements.txt. I'm fine with alignment the other way around as well 👍